### ISSUE TITLE
Format whole project

### ISSUE THREAD

**Issue by vBoykoGit** [Jintin/Swimat#223]
Is it possible?

**Comment by Jintin** [comment]
@saagarjha I remember we can use Swimat in command line right?

**Comment by saagarjha** [comment]
Yeah, we do. `swimat -r` or `find . -name *.swift | xargs swimat` might be helpful.

**Comment by vBoykoGit** [comment]
> Yeah, we do. `swimat -r` or `find . -name *.swift | xargs swimat` might be helpful.

It only formats files inside the current folder but not in the subfolders.

**Comment by saagarjha** [comment]
The former or the latter? I am fairly sure the second one should work since it's not our code doing path traversal…

**Comment by Jintin** [comment]
Do you guys think we should mention this in our README?
After that I think we ok to close it?

**Comment by vBoykoGit** [comment]
> Do you guys think we should mention this in our README?
> After that I think we ok to close it?

Absolutely, it would be very helpful.

**Comment by Jintin** [comment]
@vBoykoGit Just want to check again on the issue, does both way @saagarjha provide is working?

@saagarjha Seems you're the best people to update README on this part, would you like to help?
I think both how to install and how to use is useful for most people.

**Comment by saagarjha** [comment]
Sure, I think adding a section describing command-line usage would probably be helpful; I'll write one.

**Comment by vBoykoGit** [comment]
I got

```
 % swimat -r                          
Finished formatting 0 files.
```

But there are swift files in the directory.

**Comment by chenYtao** [comment]
swimat -r                          
Finished formatting 0 files.

**Comment by vBoykoGit** [comment]
+++

**Comment by Ehco1996** [comment]
in your project root dirctory

`swimat -r .`
